src/App.js

```diff
@@ -1,16 +1,39 @@
 import React from 'react';
 import './App.css';
+// import ChatEntry from './components/ChatEntry';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
+
 
 const App = () => {
+  const [chatEntries, setChatEntries] = useState(chatMessages)
+  const [likeCount, setLikeCount] = useState(0);
```

Comment:
Because we can determine the number of likes from `chatEntries`, `likeCount` should not be its own state variable. In general, we want state variables to not depend on one another, otherwise we get multiple sources of truth and increase the risk of bugs.